reference/test/matrix/sparsity_csr_kernels.cpp

```diff
@@ -174,6 +174,20 @@ TYPED_TEST(SparsityCsr, AppliesToDenseVector)
 }
 
 
+TYPED_TEST(SparsityCsr, AppliesToMixedDenseVector)
+{
+    using Vec = typename TestFixture::Vec;
+    using T = typename TestFixture::value_type;
```

Comment:
```suggestion
    using value_type = typename TestFixture::value_type;
    using T = gko::next_precision<value_type>;
    using Vec = gko::matrix::Dense<T>;
```
assuming you want `T` to be the new type.